### PR TITLE
dev-lua/luafilesystem: version bump to 1.7.0.2

### PR DESCRIPTION
**Pull request by vpayno**
Signed-off-by: Victor Payno <vpayno+gentoo@gmail.com>

Updated the ebuild so that it builds the lib for all the installed Lua
versions. If luajit use flag is used, it will pull the appropriate
lua5.x abi matching version from the build since it replaces it.

@zmedico @gyakovlev @chutz

### CODE REVIEW

**Comment by vpayno** [comment]
@gyakovlev this one looks ready

File: dev-lua/luafilesystem/luafilesystem-1.7.0.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+		# OS dependent
+		LIB_OPTION=-shared #for Linux
+
+		LIBNAME=$T.so.$V
+
+		# Compilation directives
+		WARN=-O2 -Wall -fPIC -W -Waggregate-return -Wcast-align -Wmissing-prototypes -Wnested-externs -Wshadow -Wwrite-strings -pedantic
+		INCS=\$(LUA_INC)
+		CFLAGS=\$(WARN) \$(INCS)
```
Comment (gyakovlev) [inline dev-lua/luafilesystem/luafilesystem-1.7.0.2.ebuild]: this omits flags set by user.

File: dev-lua/luafilesystem/luafilesystem-1.7.0.2.ebuild [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+		CC=gcc
```
Comment (gyakovlev) [inline dev-lua/luafilesystem/luafilesystem-1.7.0.2.ebuild]: this is incorrect.
should be `$(tc-getCC)`

**Comment by gyakovlev** [comment]
merged and fixed remaining issues

https://github.com/gentoo/gentoo/commit/67daf2dadfee0b73600089cac26ad359c054e967

**Pull request closed without merging**